ITS-UofIowa/sitenow_custom_profile
# Media oembed

## Testing media oembed

1. Update `system.make.yml` to use the media-oembed branch of the sitenow profile.
```yml
sitenow:
    type: profile
    download:
      url: git@github.com:ITS-UofIowa/sitenow_custom_profile.git
      branch: media-oembed
```
2. Run `nile ldu`
3. Sync a custom service site from production
4. Run database updates: `drush updb -y`
5. Check any existing YouTube embeds to ensure that they are still working.
6. Try to embed media from other providers. More info can be found at the Oembed configuration page -> Providers

## Before merging
We need to merge https://github.com/ITS-UofIowa/sitenow_media/pull/20 and create a release. Then update system.make.yml for sitenow_media with that release.
